packages/packages-groonga-org-package-task.rb

```diff
@@ -175,19 +175,32 @@ def download_packages(target_namespace)
     mkdir_p(repositories_dir)
     download_dir = "#{base_dir}/tmp/downloads/#{@version}"
     mkdir_p(download_dir)
+    
+    github_releases_base_url = "https://github.com/groonga/groonga/releases/download"
```

Comment:
`packages-groonga-org-package-task.rb` is shared with other packages that use packages.groonga.org such as Mroonga and PGroonga. So putting `groonga/groonga` here isn't a good idea.